src/c/common_internals.h

```diff
@@ -24,7 +24,7 @@ extern "C" {
 // -------------------------------------------------------------------
 //                     INTERNAL UTIL FUNCTIONS
 // -------------------------------------------------------------------
-bool check_buffer(mcBuffer* mb, const uint32_t bytes);
+bool check_buffer(ucdrBuffer* mb, const uint32_t bytes);
```

Comment:
If a user has a function with same name, it will be an error at linking time.

Reply:
Right. Fixed there and in any place where there are internal functions.